source/uploader.ts

```diff
@@ -98,11 +98,11 @@ export class Uploader {
   /** Additional data for Component entity */
   private data: CreateComponentOptions["data"];
   /** @deprecated - Remove once Session.createComponent signature is updated. */
-  createComponentResponse: CreateResponse<Data> | null;
+  createComponentResponse: CreateResponse<EntityType> | null;
   /** @deprecated - Remove once Session.createComponent signature is updated. */
   uploadMetadata: GetUploadMetadataResponse | null;
   /** @deprecated - Remove once Session.createComponent signature is updated. */
-  createComponentLocationResponse: CreateResponse<Data> | null;
+  createComponentLocationResponse: CreateResponse<EntityType> | null;
```

Comment:
should these be `CreateResponse<"Component">` and `CreateResponse<"ComponentLocation">`?

Reply:
No idea. CC @gismya.

Reply:
Sounds correct. Changing.